composer.json

```
@@ -5,7 +5,7 @@
     "license": "LGPL-3.0-only",
     "require": {
         "pocketmine/pocketmine-mp": "^5.0",
```

```suggestion
        "pocketmine/pocketmine-mp": "^5.36.0",
```

These changes depend on changes introduced in 5.36.0